chore: Add tests to README (#90)
decentraland/collections-graph: README.md

### Install

```bash
npm ci
```

### Test

Tests are defined inside the `./tests` directory and executed using:

```bash
npm test
```

> `Postgres 14` is required to run tests locally. That is because `graph-node`, the underlying engine to index the subgraph, needs it.

Reviewer: ```suggestion
> **Take into consideration:**
>`Postgres 14` is required to run tests locally because `graph-node`, the underlying engine to index the subgraph, needs it.
```

> An alternative to run tests locally without Postgres is to have `Docker` installed. In that case, tests can be run using:

Reviewer: ```suggestion
An alternative to run tests locally without installing Postgres is through `Docker. To run them, use the following command:
```

```
npm run test:docker
```

> Tests are run on PRs to master just with `npm test` because the `ubuntu-latest` image being used contains Postgres 14.

Reviewer: ```suggestion
> **CI Notice:**
> Tests are run on the CI using `npm test` because the `ubuntu-latest` image used in the CI contains Postgres 14.
```

> More about how to write subgraph tests can be found on https://thegraph.com/docs/en/developing/unit-testing-framework/

Reviewer: ```suggestion
More about how to write subgraph tests can be found on the [graph's unit testing documentation](https://thegraph.com/docs/en/developing/unit-testing-framework/).
```
